packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -364,12 +353,8 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
     });
 
     itExpects("serialize/rehydrate multiple times then attach", [
-        { eventName: "fluid:telemetry:Container:ContainerClose", error: "0x202" },
+        { eventName: "fluid:telemetry:Container:ContainerClose" },
```

Comment:
won't this still fail? I don't see where the container gets closed in the ODSP case.